spec/cob_index/suppress_config_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "traject/array_writer"
+
+RSpec.describe "Suppress configuration" do
+  let(:indexer) do
+    settings = {
+      "nokogiri.each_record_xpath" => "//oai:record",
+      "nokogiri.namespaces" => { "oai" => "http://www.openarchives.org/OAI/2.0/" },
+      "solr.url" => "http://example.com/solr",
+      "writer" => Traject::ArrayWriter.new,
+    }
+
+    indexer = CobIndex::NokogiriIndexer.new(settings)
+    indexer.load_config_file("lib/cob_index/suppress_config.rb")
+    indexer
+  end
+  let(:oai_xml) do
+    <<~EOT
+    <?xml version="1.0" encoding="UTF-8"?>
+    <OAI-PMH xmlns="http://www.openarchives.org/OAI/2.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
+      <responseDate>2020-03-03T04:16:09Z</responseDate>
+      <request verb="ListRecords" metadataPrefix="marc21" set="blacklight" from="2020-03-02T20:47:11Z">https://na02.alma.exlibrisgroup.com/view/oai/01TULI_INST/request</request>
+      <ListRecords>
+      #{record_text}
+      </ListRecords>
+    </OAI-PMH>
+    EOT
+  end
+  let(:records_file) { StringIO.new(oai_xml) }
+  let(:records) { Traject::NokogiriReader.new(records_file, []) }
+  let(:record) { records.first }
+
+  before do
+    stub_const("ENV", ENV.to_hash.merge("SOLR_URL" => "http://example.com/solr"))
+  end
+
+
+  context "OAI record should be suppressed" do
+    let(:record_text) {
+      <<~EOT
+      <record>
+        <header status="not-deleted">
+          <identifier>oai:alma.01TULI_INST:991022366369703811</identifier>
+          <datestamp>2020-03-03T03:54:35Z</datestamp>
+          <setSpec>blacklight</setSpec>
+          <setSpec>rapid_print_journals</setSpec>
+          <setSpec>blacklight_qa</setSpec>
+        </header>
+      </record>
+      EOT
+    }
+
+    it "will skip the record" do
+      expect(indexer.process_record(record).skip?).to eq(true)
+    end
+  end
+
+  context "OAI record will not be suppressed" do
```

Comment:
context is "OAI record will not be suppressed" but then the record gets processed in a way that should suppress it. 

Are the two contexts reversed?